### DISCUSSION TITLE
Title isn't showing up in my Search Results

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2495] [Q&A]
I was trying to get my search results to sort by Title, and saw on another post that you have to map the Title to a RefinableString Managed Property.
I did this and immediately went to my search page and noticed none of the title were there anymore. I assumed it was because I needed to change the Layout slot to show the RefinableString instead of the Title field. 
Once I went into the webpart and changed it, I clicked saved and nothing happened. 
I don't think this is because of the mapped property, I think it is because when I save, and go back in, my changes are not saved.
I then went and unmapped Title from the refinablestring hoping it would fix it? It did not.
So then I searched for other possible solutions. I saw a lot of suggestions of using the beta release to fix things in your issues/discussions so I tried that. This also did not help.
This morning I then tried reindexing my site, and it has been over 6 hours.. although it is not fixed, there are a few coming through. 
I am not so sure what else to do.

**Comment by nicolesherman00** [comment]
![image](https://user-images.githubusercontent.com/22782498/198852234-e7f8439c-c683-4c87-a245-88c04ad91533.png)
And as you can see in our doc library we have checked and made sure there is a title in all documents..
![image](https://user-images.githubusercontent.com/22782498/198852399-4071d37c-14dc-412b-917f-65cf8d0eddee.png)

**Reply by wobba** [reply]
You have to map the crawled property `Office:2` to get the SP title.

**Reply by nicolesherman00** [reply]
There is no such thing in my environment.
![image](https://user-images.githubusercontent.com/22782498/199824475-306db914-cefd-43b6-a49e-2ee8f19700c0.png)

And do you mean map Office:2 to a refinablestring?

Thanks

**Reply by wobba** [reply]
Pick category "office", then search and you will see 2 listed.
![image](https://user-images.githubusercontent.com/51104/200186074-fbd5ad1e-4ce6-4815-8cfc-24a9521e6f12.png)
